console/cypress/helpers/remoteSchemaHelpers.ts

```diff
@@ -1,18 +1,30 @@
 import { ADMIN_SECRET_HEADER_KEY } from '../../src/constants';
 
 export const baseUrl = Cypress.config('baseUrl');
-export const getRemoteSchemaName = (i, schemaName) =>
+
+export const getRemoteSchemaName = (i: number, schemaName: string) =>
   `test-remote-schema-${schemaName}-${i}`;
+
 export const getRemoteGraphQLURL = () =>
   'https://realtime-poll.demo.hasura.app/v1/graphql';
```

Comment:
@beerose This server is not active anymore so some of the tests are failing. Can I get a server that is active for the cypress test?

Reply:
The server is live but creating remote-schema is failing